### DISCUSSION TITLE
v0.11.0

### DISCUSSION THREAD

**Discussion by Blinue** [Blinue/Magpie#918] [Announcements]
https://github.com/Blinue/Magpie/compare/v0.10.6...v0.11.0

<hr /><em>This discussion was created from the release <a href='https://github.com/Blinue/Magpie/releases/tag/v0.11.0'>v0.11.0</a>.</em>

**Comment by Craxuan** [comment]
Excuse me, I don't get how to use a shipped effect. Under scaling modes I noticed that I can add effects for scaling modes, but I'm not sure if it's working at all. Like, let's say I added 3 effects to Anime4K. Which one does the game chooses to use? The topmost one? How do I specify it for a specific game? Thanks.

**Reply by hauuau** [reply]
Each scaling mode is a chain of effects. Effects in a scaling mode are applied sequentially.
If you want to use different scaling mode per game then you need to create a profile for that game and choose specific scaling mode there. By default the default profile is being used and default scaling mode is selected there.
If you want to check which effects are used you could use the in-game overlay hotkey to display an overlay which will report effects which are being used currently.

**Reply by Craxuan** [reply]
I see! I already did 2/3 of what you said before I posted this question, I know how to change scaling modes, but the effects are a different story. Specifically, I couldn't figure out how to apply the effects (or if they're being applied at all for the matter, some effects barely look any different from another), but with the in-game overlay I can tell for sure that it's actually being applied now (and some experimentation). Thank you!

**Comment by Bluelagoon1** [comment]
Hey i have question about what kind of FSR version does magpie uses, is it 2.0 3.0 ? Or it has nothing about the actual AMD FSR one, thanks.

**Reply by Blinue** [reply]
It's FSR1.

**Reply by Bluelagoon1** [reply]
Would that be possible to add fsr3 on magpie soon since amd gave the source code freely ?

**Reply by Blinue** [reply]
FSR 2/3 can only be integrated by game developers #395 https://github.com/Blinue/Magpie/discussions/284#discussioncomment-2400536 #815
